glide.lock

```diff
@@ -1,215 +1,16 @@
-hash: 9589cf38139f18ca918955308856f05f0f5196ca6189cd6961ffd1aff19d8ba6
```

Comment:
@jimmidyson I probably messed up the glide lock file (did a 'glide get  github.com/stretchr/testify/assert' for the test).

What's the canonical way to add a new dependency ?